store/src/java/com/zimbra/cs/service/util/JWTUtil.java

```diff
@@ -136,6 +136,10 @@ public static Claims validateJWT(String jwt, String salts) throws ServiceExcepti
             if (acct == null ) {
                 throw AccountServiceException.NO_SUCH_ACCOUNT(claims.getSubject());
             }
+            if (acct.hasInvalidJWTokens(jti)) {
+                ZimbraLog.account.debug("jwt: %s is no longer valid, has been invalidated on logout", jti);
+                throw AuthFailedServiceException.AUTH_FAILED("Token not valid");
```

Comment:
Add the reason for invalid token, that it is invalidated.

Reply:
done